Documentation/compatibility/workflow-designer-accessibility-4-8.md

```diff
@@ -0,0 +1,56 @@
+## Accessibility improvements in Windows Workflow Foundation (WF) workflow designer
+
+### Scope
+Minor
+
+### Version Introduced
+4.8
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+The Windows Workflow Foundation (WF) workflow designer is improving how it works with accessibility technologies. These improvements include the following changes:
+- Users using Narrator will see improvements in FlowSwitch case labels.
+- Users who choose High Contrast themes will see many improvements in the visibility of the Workflow Designer and its controls like better contrast ratios between elements and more noticeable selection boxes used for focus elements.
+- Users using Narrator will see improvements in button descriptions.
+
+- [x] Quirked
+- [ ] Build-time break
+
+### Recommended Action
+If you have an application with a re-hosted workflow designer, your application can benefit from these changes by performing either of these actions:
+- Recompile your application to target the .NET Framework 4.7.1. These accessibility changes are enabled by default.
```

Comment:
Did you mean .NET Framework 4.7.1? The switch (`Switch.UseLegacyAccessibilityFeatures.3`) is the .NET Framework 4.8 switch, isn't it? Or is the switch enabled if applications are running on 4.8 and targeting 4.7.1 or later?
```suggestion
- Recompile your application to target the .NET Framework 4.7.1 or later. These accessibility changes are enabled by default.
```

Reply:
Done.